GUIDELINES.md

```diff
@@ -0,0 +1,95 @@
+# DIP writing guidelines
+
+## Expected content
+
+Writing a good informative DIP with solid chances to get approved is very
+hard work - it is often underestimated how much effort is required. Good
+improvement proposal should:
+
+1. Explain the value gained from implementing the change. Alternative solutions
+   must be considered and compared.
+2. Use real-world example, either taken from existing projects, or looking
+   like ones.
+3. Consider possible objections against the proposal and research them
+   as part of DIP document.
+4. Stick to formal and technical language in general, avoid colloquialisms. A
+   DIP should be formulated like a scientific paper, as opposed to an op-ed
+   article.
+5. Provide sufficient techniqal details that can be used as a specification
+   by D compiler developers.
+
+Be prepared for a lot of work. There are always many ideas proposed but
+many fewer developers commited to pursuing the idea to the final stages of
```

Comment:
s/many fewer/much fewer/ 

(an adverb, not an adjective should modify another adjective); or

s/many fewer/far fewer/

(a classic idiom); or simply

s/many fewer/fewer/